data/test/scenarios/wml_tests/UnitsWML/special_note_from_movetype.cfg

```diff
@@ -1,7 +1,7 @@
 # wmllint: no translatables
 
 #####
-# API(s) being tested: [movetype][special_note],[store_unit],[unstore_unit]
+# API(s) being tested: [movetype][special_note]
```

Comment:
This was part of a bugfix of `[store_unit]`.

Reply:
Same comment as previously where this should be just `[store_unit]` instead.

Reply:
Changed to `[store_unit]`.